css/styles.css

```diff
@@ -86,15 +84,19 @@ main {
   padding: 30px;
 }
 
-main input {
+input {
+  font-family: 'Roboto', sans-serif;
   font-size: 20px;
   background: #fff;
   border: 1px solid #111;
-  padding: 0px 30px;
-  width: 400px;
-  font-family: helvetica, arial, sans-serif;
-  height: 72px;
-  line-height: 74px;
+  padding: 0px 20px;
```

Comment:
Watch the padding on the sides here - on mobile it makes the available space for typing quite small.